WorkspaceServer/Kernel/CSharpKernel.cs

```diff
@@ -46,6 +48,7 @@ public CSharpKernel()
             _metadataReferences = ImmutableArray<MetadataReference>.Empty;
             SetupScriptOptions();
             Name = KernelName;
+            ExtensionsPath = new RelativeDirectoryPath("interactive-extensions/cs");
```

Comment:
Will both scripts and assemblies be under this path?

Reply:
I renamed it to say "AssemblyExtensionsPath"